components/coaching/AboutSCFooter.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <section class="container mx-auto px-6 md:px-8 mt-[83px]">
+    <div class="flex flex-wrap xl:flex-nowrap items-center justify-center gap-y-4">
+      <div class="SC-text">
+        {{ pageData.fields.paragraph }}
+      </div>
```

Comment:
This will need to use the RichText component as it is a RichText field from Contentful